guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc

```diff
@@ -8,7 +8,7 @@ If you plan to use {Project} on AWS, ensure that the use case scenarios that you
 
 You can perform the following {ProjectName} use cases on AWS:
 
-ifndef::foreman-deb[]
+ifdef::katello,orcharhino,satellite[]
 * {ContentManagementDocURL}Managing_Red_Hat_Subscriptions_content-management[Managing Red Hat Subscriptions]
 * {ContentManagementDocURL}Importing_Content_content-management[Importing Content]
 * {ContentManagementDocURL}Managing_Errata[Managing Errata]
```

Comment:
Note that the line below this (Registering a Host Manually) does not link to anything any anymore. Is that because it's not using xref so it's not checked?